Sources/Commands/SwiftTestCommand.swift

```diff
@@ -709,6 +720,21 @@ extension SwiftTestCommand {
     }
 }
 
+fileprivate extension Triple {
+    var llvmCovArchArgument: String? {
+        guard let arch = self.arch else {
+            return nil
+        }
+        switch arch {
+        case .aarch64:
```

Comment:
This is unfortunate but probably the right thing to do for now. For completeness, we might want to apply it to Apple platforms beyond macOS